### ISSUE TITLE
Enhancing Greek Localization

### ISSUE THREAD

**Issue by vtalos** [commons-app/apps-android-commons#5683]
### What is the user problem or growth opportunity you want to see solved?

Several Greek IDs are missing from the strings.xml file in the values-el file.

### How do you know that this problem exists today? Why is this important?

Missing Greek IDs can lead to a subpar user experience for Greek-speaking users.

### Who will benefit from it?

Greek users will get an upgraded experience.

### Anything else you would like to add?

I would like to work on this issue with @karyotakisg.

**Comment by karyotakisg** [comment]
Indeed, that  addition could be valuable, as sections of the app like "settings" lack proper translation.

**Comment by nicolas-raoul** [comment]
Thanks a lot for your interest!
Here is the webapp where Greek localization can be performed:

https://translatewiki.net/w/i.php?title=Special:Translate&group=commons-android-strings&filter=%21translated&action=translate&language=el

This webapp overwrites strings.xml/etc regularly.
